packages/ui/src/memberships/constants/externalResourceLink.ts

```diff
@@ -0,0 +1,8 @@
+export const ExternalResourceLink = {
+  TELEGRAM: 'https://web.telegram.org/k/#@',
```

Comment:
I don't know much about Telegram but @ivanturlakov suggested to change this to:
```suggestion
  TELEGRAM: 'https://t.me/',
```